src/AspNetCoreModuleV2/OutOfProcessRequestHandler/dllmain.cxx

```diff
@@ -21,6 +21,8 @@ IHttpServer *       g_pHttpServer = NULL;
 HINSTANCE           g_hWinHttpModule;
 HINSTANCE           g_hAspNetCoreModule;
 HANDLE              g_hEventLog = NULL;
+FILE_WATCHER*       g_pFileWatcher = NULL;
```

Comment:
Why do you need it globally? Can't it be per `application` object?

Reply:
You don't want to create so many file watcher threads. A global one, single watcher thread